src/Microsoft.DotNet.Cli.Utils/CommandResolution/PublishedPathCommandResolver.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class PublishedPathCommandResolver : ICommandResolver
```

Comment:
So then how do we expect "dotnet test" to work?  Does it need to publish your test project first?

I would think "dotnet test" only needs to "build" the project, and it should work.